src/WpBookingsUiModule.php

```diff
@@ -170,16 +169,12 @@ protected function _isOnBookingsPage()
      */
     protected function _getBookingsAppState($c)
     {
-        /* @var ControllerInterface $controller */
-        $controller = $c->get('eddbk_services_controller');
-        $services = iterator_to_array($controller->get());
-
         return [
             /*
              * All available statuses in application.
              */
-            'statuses' => $this->_trigger('eddbk_bookings_statuses', [
-                'statuses' => $this->_getTranslatedStatuses($c)
+            'statuses' => $this->_trigger('eddbk_bookings_translated_statuses', [

```

Comment:
This looks weird. The app state will be like this?

Reply:
No man, as you remember, this is the event (you can see in the line's end `...->getParam('statuses')`. So state will be like this:
```
{
  'statuses': {
    ...
    'draft': 'Draft',
    ...
  }
}

Reply:
Yep, true, my bad.